### PR TITLE
Added menu

### PR DESCRIPTION
**Pull request by MouliTadinada**

### CODE REVIEW

File: Diary.c [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    This method asks for input and provides the output
+    blah blah blah
+    */
+    char ch;
+    printf("\t\t******[SECURE PERSONAL DIARY]*****\n\n");
+    printf("\t\t1. LOGIN");
+    printf("\t\t2. SIGN-UP\n\n");
+    printf("\t\tENTER YOUR CHOICE: ");
+    ch=getche();
+    return ch-48;
```
Comment (souravabhijith) [inline Diary.c]: What is 48 ? Use char notation .

File: Diary.c [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    /*
+    This method asks for input and provides the output
+    blah blah blah
+    */
+    char ch;
+    printf("\t\t******[SECURE PERSONAL DIARY]*****\n\n");
+    printf("\t\t1. LOGIN");
+    printf("\t\t2. SIGN-UP\n\n");
+    printf("\t\tENTER YOUR CHOICE: ");
+    ch=getche();
```
Comment (souravabhijith) [inline Diary.c]: Make input and output seperate methods

**Comment by souravabhijith** [comment]
Add gitignore file , No need to commit exe and .o files .

File: Diary.c (hunk 1 above)
Comment (MouliTadinada) [inline Diary.c]: changed !!

**Pull request merged**